mod/color_style.hpp

```diff
@@ -33,4 +33,92 @@ class diwako_dui_colors
         blue = "#3333FF";
         yellow = "#D9D900";
     };
+
+    class cblind:protanopia

```

Comment:
classname cblind is defined mulstiple times, this won't work.

Also inherit from "standard" instead.